Main.java

```diff
@@ -1,21 +1,20 @@
 class Main {
   public static void main(String[] args) {
-   // kindly use appropriate data types for your declaration
-   // declare variables days the debtor defaulted, amount the debtor is to pay per day, amount the debtor is to pay and assign values to them
-   
-   //Declare the variable of the total amount the debtor is to pay, the days the debtor paid for, amount the debtor is left to pay, days the debtor has not payed for
+    long daysTheDebtor = 60;
+    long amountPayPerDay = 100;
+    long amountPaid =2700;
```

Comment:
Long is actually inappropriate for these variables. You should have used byte for the days and float for the amounts